wemake_python_styleguide/visitors/ast/annotations.py

```diff
@@ -42,13 +44,29 @@ def visit_arg(self, node: ast.arg) -> None:
         self._check_arg_annotation(node)
         self.generic_visit(node)
 
+    def _check_for_literal_none(self, node: ast.Subscript) -> None:
```

Comment:
As I already said: let's move this logic into `visitors/ast/annotations.py`

Reply:
Done, please check the freshest commit. :)